history.adoc

```diff
@@ -7,6 +7,7 @@
 
 === Working version (most recent first)
 
+* {issues}141[Issue #141]: Clarification that text-valued variables and attributes can be Unicode vlen strings or UTF-8 char arrays.
```

Comment:
* {issues}141[Issue #141]: Clarification that text-valued variables and attributes can be UTF-8 encoded vlen strings or char arrays.